base/pkg/cache.jl

```diff
@@ -2,7 +2,7 @@
 
 module Cache
 
-import ...LibGit2, ..Dir, ...Pkg.PkgError
+import ...LibGit2, ..Dir, ...Pkg.PkgError, ..Git
```

Comment:
Do not use `Git` module. It will be removed. LibGit2 has normalization function as well.